src/main/java/org/jenkinsci/plugins/secure_requester_whitelist/Whitelist.java

```diff
@@ -30,19 +30,25 @@
 import jenkins.model.GlobalConfigurationCategory;
 import jenkins.model.Jenkins;
 import net.sf.json.JSONObject;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
 import org.kohsuke.stapler.StaplerRequest;
 
-@Extension public class Whitelist extends GlobalConfiguration {
+@Extension
+@Symbol("secure_requester_whitelist")
```

Comment:
Why did you have to add the Symbol now? is it needed for JCasC, good practice, other reason?

Reply:
The symbol could be used by JCasC as a facilitator, not strictly required but largely encouraged I would say :)

Reply:
:+1: Good to know!

Reply:
```suggestion
@Symbol("secureRequesterWhitelist")
```

Just to follow the standard naming pattern in JCASc

Reply:
Yes, it is needed for JCasC. I haven't tried without so I don't know what JCasC does when there is no symbol. Pipeline tries to guess based on class name and in that case it would be `Whitelist` which is not very descriptive for the plugin.